devutils/develop-libraries.bat

```diff
@@ -0,0 +1,49 @@
+@echo off
+pushd .
+
+SET scriptPath=%~dp0
+SET scriptPath=%scriptPath:~0,-1%
+cd /D %scriptPath%
+
+:: Get RCC
+IF NOT EXIST ".\rcc.exe" (
+    curl -o rcc.exe https://downloads.robocorp.com/rcc/releases/latest/windows64/rcc.exe --fail || goto :error
+)
+
+:: Go to repo root
+cd ..
+
+:: Check if .venv folder exists
+IF EXIST ".\venv" (
+    echo Detected existing developement environment.
+    echo Do you want to create a clean environment? [Y/N]
+    choice /C YN /N /M "Select Y for Yes (clean environment) or N for No (use existing):"
+    IF ERRORLEVEL 2 GOTO USE_EXISTING
+)
+
+rcc venv dev-env-libraries.yaml -s dev-library-env --force
+
+::INSTALL_DEPENDENCIES
+call .\venv\Scripts\activate.bat
+
+pip install poetry==1.7 invoke==2.2
```

Comment:
I would recommend installing these from our own [requirements](https://github.com/robocorp/robocorp/blob/master/devutils/requirements.txt) list (single source of truth), which is what we also use in CI.

Reply:
`pip>=23.2.1` in the requirements.txt is really bad, there have been multiple breaking changes to pip since 23.2.1.
23.3 was breaking and now 24 is also out. Locking these down is really crucial, especially if this affects the released libraries.